tables/basic-table/step2.md

```diff
@@ -0,0 +1,53 @@
+In this exercise, we'll create a basic table component with standard rows and columns (cells) definitions.
+
+Your columns can be expressed as an array of strings or an array of objects which match the type "ICell". The columns definition is passed to the `cells` prop of the Table component.
+
+Your rows can be expressed as a multidimentional array of strings, or as an array of objects which match the type "IRow". The rows definition is passed to the `rows` prop of the Table or TableBody component.
+
+## Task
+
+1) Navigate to the `src` folder and open `App.js`
+
+2) Next, we'll set up the general structure of the table and the necessary imports.
+
+Copy the following code into the App.js file, replacing all of the content there:
+
+<pre class="file" data-filename="App.js" data-target="replace">
+import * as React from &quot;react&quot;;
+import * as ReactDOM from &quot;react-dom&quot;;
+import { Table, TableHeader, TableBody } from &quot;@patternfly/react-table&quot;;
+
+const BasicTableComponent = () =&gt; {
+  return (
+    &lt;Table caption=&quot;Basic Patternfly React Table&quot; cells={[]} rows={[]}&gt;
+      &lt;TableHeader /&gt;
+      &lt;TableBody /&gt;
+    &lt;/Table&gt;
+  );
+};
+ReactDOM.render(&lt;BasicTableComponent /&gt;, document.getElementById(&quot;root&quot;));
+</pre>
+
+3) Now, replace the empty arrays with simple row/column definitions.
+
+<pre class="file" data-filename="App.js" data-target="clipboard">
+const columns = [&quot;First column&quot;, &quot;Second column&quot;, &quot;Third column&quot;];
+const defaultRows = [
+  [&quot;Row 1 column 1&quot;, &quot;Row 1 column 2&quot;, &quot;Row 1 column 3&quot;],
+  [&quot;Row 2 column 1&quot;, &quot;Row 2 column 2&quot;, &quot;Row 2 column 3&quot;],
+  [&quot;Row 3 column 1&quot;, &quot;Row 3 column 2&quot;, &quot;Row 3 column 3&quot;]
+];
+</pre>
```

Comment:
I split these instructions up a bit more, is it enough? I like the idea of having the user add these for the first time because we can have tables with no rows or columns technically, even though this doesn't do much. No problem preloading the table though if you think that makes for a better flow.